### PR TITLE
fix: csv templates download not working

### LINKED ISSUE #568: Bug: SMP mix template not downloading
**Describe the Bug**
Under the "Calculation of SMP Mix" screen, the link to "Download calculation of SMP mix template", is not working. There is a new page that opens to this url: https://nr-spar-test-frontend.apps.silver.devops.gov.bc.ca/downloads/SMP_Mix_Volume_template.csv
However a template does not actually download.

Image

**Expected Behaviour**
A .csv template for SMP mix appears in your browser downloads

**Steps To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Calculation of SMP Mix'
2. Click on 'Download calculation of SMP mix template link'
3. See blank url

**Screenshots**
See above

### PR DESCRIPTION
**Pull request by RMCampos**
# Description
Closes #568 

#### New
- None

#### Changed
- CSV template file locations, moved to src folder to be able to import them.
- Function responsible for fetching the file location.

#### Removed
- None

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

No post-deployment tasks needed.

###  What gif/image best describes this PR or how it makes you feel?

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-31-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-31-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-31-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by mgaseta — APPROVED** [review]
Works like a charm!
Good job! :100: :1st_place_medal:

File: frontend/src/components/SeedlotRegistrationSteps/ParentTreeStep/constants.tsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ import {
   InfoSectionConfigType, FileConfigType
 } from './definitions';
 
+import SeedlotCsvTemplate from '../../../assets/csv/Seedlot_composition_template.csv?url';
+import SmpMixCsvTemplate from '../../../assets/csv/SMP_Mix_Volume_template.csv?url';
+
```
Comment (craigyu) [inline frontend/src/components/SeedlotRegistrationSteps/ParentTreeStep/constants.tsx:11]: Curious on what the `?url` does?

**Review by craigyu — APPROVED** [review]
Awesome!

File: frontend/src/components/SeedlotRegistrationSteps/ParentTreeStep/constants.tsx (hunk 1 above)
Comment (craigyu) [reply frontend/src/components/SeedlotRegistrationSteps/ParentTreeStep/constants.tsx:11]: Ricardo found it here :
🪄
https://github.com/vitejs/vite/discussions/8271#discussioncomment-5192948

Comment (RMCampos) [reply frontend/src/components/SeedlotRegistrationSteps/ParentTreeStep/constants.tsx:11]: That's a great question my friend! I've found this on a vitejs discussion thread: https://github.com/vitejs/vite/discussions/8271#discussioncomment-5192948

**Pull request merged**